<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPINHYDW0993" ignored="0" name="FailedTests" tests="6" failures="6" timestamp="01 Jun 2017 16:53:24 GMT" time="0.009" errors="0">
  <testcase name="cosineOfDouble" time="0.001" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.cosineOfDouble(FailedTests.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- cosineOfDouble -->
  <testcase name="cotangentofDouble" time="0.0" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.cotangentofDouble(FailedTests.java:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- cotangentofDouble -->
  <testcase name="divideByZero" time="0.0" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.divideByZero(FailedTests.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- divideByZero -->
  <testcase name="powerOfDouble" time="0.0" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.powerOfDouble(FailedTests.java:24)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- powerOfDouble -->
  <testcase name="productOfDouble" time="0.001" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.productOfDouble(FailedTests.java:16)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- productOfDouble -->
  <testcase name="tangentofDouble" time="0.0" classname="com.epam.calculatorTests.FailedTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.epam.calculatorTests.FailedTests.tangentofDouble(FailedTests.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- tangentofDouble -->
</testsuite> <!-- FailedTests -->
